lighthouse-cli/test/fixtures/ric-shim.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+<body>
+Request Idle Callback Shim Smoke Test
+<script>
+  let count = 0;
+  const iterations = 40;
+
+  if (window.location.search.includes('short')) {
+    window.ricTaskDuration = 1;
```

Comment:
If i remember correctly let/const dont move up into the global scope if they are in some sort of curly braces but i could do something like this, wdyt? Or just use var if you prefer that because that should just move up.

let ricTaskDuration;
if (window.location.search.includes('short')) {
    ricTaskDuration = 1;
  }
  if (window.location.search.includes('long')) {
    ricTaskDuration = 50;
  }

Reply:
yeah sorry that's what I meant, declaring `let ricTaskDuration = 0` up with `iterations` and then overriding it if the query param matches